### PR TITLE
Explain associative arrays without templates and lambdas

### PR DESCRIPTION
**Pull request by wilzbach**
As discussed in #335.

### CODE REVIEW

File: public/content/en/basics/associative-arrays.md [hunk 1]
```diff
@@ -66,10 +66,7 @@ int[string] wordCount(string text)
     alias pred = c => c == ' ' || c == '.'
```
Comment (stonemaster) [inline public/content/en/basics/associative-arrays.md]: Pred is obselete right?

**Comment by stonemaster** [comment]
Great, small changes but more comprehensile. After all we want to show associative arrays..

**Comment by wilzbach** [comment]
> Pred is obselete right?

Oh yes - thanks for catching that ;-)

**Comment by stonemaster** [comment]
:+1:

**Pull request merged**